lib/Service/VerdictService.php

```diff
@@ -255,4 +248,27 @@ private function createAndConnectVaas(): Vaas {
 		$vaas->Connect($this->authenticator->getToken());
 		return $vaas;
 	}
+
+    /**
+     * Checks if the file is in the blocklist or not in the allowlist and throws an exception if it is not allowed to scan the file.
+     * @param string $filePath
+     * @return bool
+     */
+    public function isAllowedByAllowAndBlocklist(string $filePath): bool {
+        $blocklist = $this->getBlocklist();
+        $this->logger->debug("Blocklist: " . implode(", ", $blocklist));
+        foreach ($blocklist as $blocklistItem) {
+            if (str_contains(strtolower($filePath), strtolower($blocklistItem))) {
```

Comment:
What pattern are we intended to use in this field? all substrings, prefixes (for pathes)?

mypath for example will match on

/mypath/file.php but also on
/path/mypath.php